javascript-functions/main.js

```diff
@@ -0,0 +1,43 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult =
+convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function name(Hey, Beavis) {
+  var name = 'Hey, Beavis';
+  return name;
+}
+
+var nameResult = name('Beavis');
+console.log('nameResult:', nameResult);
+
+function getArea(width, height) {
+  var multiply = width * height;
+  return multiply;
+}
+
+var getAreaResult =
+getArea(17, 42);
+console.log('getAreaResult:', getAreaResult);
+
+function getFirstName(person) {
+  var getFirstName = 'Lelouche';
+  return getFirstName;
+}
+
+var getFirstNameResult =
+getFirstName({ firstName: 'Lelouche', lastName: 'Lamperouge' });
+console.log('getFirstNameResult:', getFirstNameResult);
+
+function getLastElement(Array) {
+  var getLastElement = Array;
+  return getLastElement[3];
```

Comment:
What if we call this function with `getLastElement([1, 2, 3, 4, 5, 6, 7, 8])` then it returns `4` instead of `8`